### PR TITLE
doc: update must gather to include open shift details

### PR DESCRIPTION
**Pull request by prb112**
Signed-off-by: Paul Bastide <pbastide@us.ibm.com>

### CODE REVIEW

**Review by JohnTimm — APPROVED** [review]
LGTM

**Review by ldettman — APPROVED** [review]
Looks good.

File: docs/src/pages/MustGather.md [hunk 1]
```diff
@@ -35,3 +35,101 @@ The IBM FHIR Server - Must-Gather Information
...
+1. To obtain your cluster ID using the OpenShift CLI (oc), run the following command:
+
+```
+oc get clusterversion -o jsonpath='{.items[].spec.clusterID}{"\n"}'
+```
+
+1. Provide the Custom Resource(CR) .yaml file used by the operator to configure the environment
+
+```
+kubectl get icp4acluster -o yaml > config.yaml
```
Comment (tbieste) [inline docs/src/pages/MustGather.md:112]: What is icp4acluster?

Comment (prb112) [reply docs/src/pages/MustGather.md:112]: Thanks, I'll clean that up

**Pull request merged**